docs/src/main/paradox/cassandra.md

```diff
@@ -0,0 +1,90 @@
+# Cassandra Connector
+
+The Cassandra connector provides a way to transform a Cassandra's ResultSet into a Akka Stream's Source.
```

Comment:
"The Cassandra connector provides a way to provide the result of a Cassandra query as a stream of rows", currently it sounds like you fetch the results into a `ResultSet` from the database and then stream rows from that, but the whole point is bounded memory, no?